Exif-Grabber.py

```diff
@@ -11,22 +11,23 @@
 
 # Extract exif data from picture with two methods
 # Coded By karim Shoair | D4Vinci
-# All Copyright To Squnity Company Team & Deveolopers
-"""
-photo=raw_input("\nPhoto Name : ")
-print "\n[1] Method one\n"
+# Updated to python 3 by Brink2three/BrinkGG
```

Comment:
No need to add your name; people can see Git commits and know who made them.
If you remove your name, I will accept the PR.